website/frontend/src/components/blocks/MetaForm/MetaForm.tsx

```diff
@@ -57,8 +58,11 @@ export const MetaForm: VFC<Props> = ({ programName, programHash }) => {
   const handleFilesUpload = useCallback(
     async (file) => {
       try {
-        const fileBuffer: any = await readFileAsync(file);
+        const fileBuffer: Buffer = (await readFileAsync(file)) as Buffer;
```

Comment:
```suggestion
        const fileBuffer = (await readFileAsync(file)) as Buffer;
```
adding type annotation is redundant, because type will be infer